CHANGELOG.md

```diff
@@ -21,6 +21,85 @@ After this, `Decidim::Proposals::ProposalEndorsement` and the corresponding coun
 
 ### Added
 
+- **decidim-forms**: Allow to view and export in PDF responses to surveys directly in the admin [\#5770](https://github.com/decidim/decidim/pull/5770)
```

Comment:
Can you clean the changelog please? since the start of the release of v0.22 there's no need anymore to add changelog entries :smile:

Reply:
Sure!